packages/runtime/container-runtime/src/dataStore.ts

```diff
@@ -151,6 +151,17 @@ class DataStore implements IDataStore {
         return this.fluidDataStoreChannel.request(request);
     }
 
+    /**
+     * Handle to the data store. Use this as the primary way of interacting with it, and only fall back to
+     * requesting the root object through the request pattern if the handle is not defined.
+     */
+    public get IFluidHandle(): IFluidHandle<FluidObject> | undefined {
+        // Currently treating the data store channel as a FluidObject<IFluidHandle>. If we later
+        // make changes so that it exposes the handle explicitly, we can simplify this code.
+        const maybeHandle: FluidObject<IFluidHandle<FluidObject>> = (this.fluidDataStoreChannel as any);
```

Comment:
```suggestion
        const maybeHandle = (this.fluidDataStoreChannel as FluidObject<IFluidHandle<FluidObject>>);
```
if we must cast, we should avoid the any, just cast it to what we need

Reply:
Updated locally, coming in next push.